compiler/tools/suggest.nim

```diff
@@ -200,10 +203,10 @@ proc symToSuggest(g: ModuleGraph; s: PSym, isLocal: bool, section: IdeCmd, info:
   result.filePath = toFullPath(g.config, infox)
   result.line = toLinenumber(infox)
   result.column = toColumn(infox)
-  result.tokenLen = if section != ideHighlight:
-                      s.name.s.len
-                    else:
-                      getTokenLenFromSource(g.config, s.name.s, infox)
+  if section in {ideHighlight}:
+    result.tokenLen = getTokenLenFromSource(g.config, s.name.s, infox)
+  else:
+    result.tokenLen = s.name.s.len
```

Comment:
it makes line longer and this processing may apply to other command in future.

Reply:
I understand the line length concern (the `if` expression usually goes on the next line for this reason), but tentatively changing an otherwise unrelated line only adds noise and complicates traversing the git history.

Reply:
```suggestion
  result.tokenLen = if section != ideHighlight:
                      s.name.s.len
                    else:
                      getTokenLenFromSource(g.config, s.name.s, infox)
```
This should revert the style change.